clang/include/clang/Basic/Attr.td

```diff
@@ -1451,7 +1451,7 @@ def LoopUnrollHint : StmtAttr {
 def IntelReqdSubGroupSize: InheritableAttr {
   let Spellings = [
     GNU<"intel_reqd_sub_group_size">, CXX11<"intel", "reqd_sub_group_size">,
-    CXX11<"intel", "sub_group_size"> // SYCL2020 spelling.
+    CXX11<"intel", "sub_group_size">, CXX11<"sycl", "reqd_sub_group_size">
```

Comment:
Can you file an issue to look into whether `[intel::whatever]` should be deprecated?

Reply:
I can if you'd like, but I think what makes sense there is to leave the `[[intel::whatever]]` spellings as undeprecated and available in whatever modes they make sense to support in, even if it's an identical spelling to another attribute. WDYT?

Reply:
>>but I think what makes sense there is to leave the [[intel::whatever]] spellings as undeprecated and available in whatever modes they make sense to support in, even if it's an identical spelling to another attribute.

Yes, i agree.

FYI, CXX11<"intel", "sub_group_size">- this is new attribute that we added recently and this is part of CXX11<"intel", "reqd_sub_group_size">,